src/components/DateTimePicker/_date-time-picker.scss

```diff
@@ -181,6 +181,10 @@
       }
     }
   }
+
+  .#{$iot-prefix}--date-time-picker__box__light {
```

Comment:
I think light is a modifier so according to BEM we should probably write `.#{$iot-prefix}--date-time-picker__box--light`